tests/images/debian-systemd/files/bootstrap.sh

```diff
@@ -543,7 +543,7 @@ install_via_local_files() {
     find_then_install_deb "$INSTALL_SOURCEDIR" "tedge[_-]mapper_*_$ARCH.deb"
     find_then_install_deb "$INSTALL_SOURCEDIR" "tedge[_-]agent_*_$ARCH.deb"
     find_then_install_deb "$INSTALL_SOURCEDIR" "tedge[_-]apt[_-]plugin_*_$ARCH.deb"
-    find_then_install_deb "$INSTALL_SOURCEDIR" "c8y[_-]configuration[_-]plugin_*_$ARCH.deb"
+    find_then_install_deb "$INSTALL_SOURCEDIR" "tedge-configuration-plugin_*_$ARCH.deb"
     find_then_install_deb "$INSTALL_SOURCEDIR" "tedge[_-]log[_-]plugin_*_$ARCH.deb"
```

Comment:
Out of this PR's scope, but we can simplify this line as well.
```suggestion
    find_then_install_deb "$INSTALL_SOURCEDIR" "tedge-log-plugin_*_$ARCH.deb"
```